packages/shared/lib/core/network/constants/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './official-node-urls.constant'
+export * from './base-token.constant'
```

Comment:
These should be alphabetized.

Reply:
There was a few issues with some types / enums used with each other not exporting in the correct order